_posts/2019-09-04-pulp-3-performance-blog.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Pulp 3 Performance Blog
+author: Imaanpreet Kaur
+team: Red Hat Performance Team
+members: Pradeep Surisetty, Jan Hutař, Imaanpreet Kaur
+---
+
+## Motivation
+
+As we all know, Pulp3 is under active development. So, it's predominant  to check the performance of the pulp3. Pulp team come up with a test plan and Red Hat Performance team tested these test cases. So, let’s discuss about the pulp3 performance in brief.
+
+## Test environment
+
+Pulp was installed on one CentOS7 (CentOS Linux release 7.5.1804 (Core)) system using this [playbook](https://github.com/pulp/ansible-pulp/blob/master/example-use/playbook.yml). 
+
+##### pulp version
+
+![version](https://user-images.githubusercontent.com/32102000/64325263-1a1c9380-cfc8-11e9-809c-210229c85887.png)
+
+In order to monitor the pulp3 performance, grafana monitoring has been done with the help of custom [script](https://github.com/redhat-performance/satellite-monitoring/blob/master/adhoc-scripts/get_stats_from_grafana.py). 
+
+##### grafana dashboard
+
+![grafana](https://user-images.githubusercontent.com/32102000/64325286-20ab0b00-cfc8-11e9-8ae4-0c92a475636b.png)
```

Comment:
I'm concerned the size is not readable and the graph is so cool people will want to read it. Maybe we should disinclude the graphic for that reason.